flow/flow.proto

```diff
@@ -43,6 +43,8 @@ message FlowLayer {
   string A = 3;
   string B = 4;
   int64 ID = 5;
+  int64 RTT = 6;
+  bool RTTSet = 7;
```

Comment:
Not sure you need a boolean to keep track of this.

What about comparing RTTSet to 0 (or -1 if you want to handle the case of RTT in the same nanosecond)